### PR TITLE
support for ddnss.de

### PR DESCRIPTION
**Pull request by helbgd**

### CODE REVIEW

**Comment by Neilpang** [comment]
please check this guide first:
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

**Comment by helbgd** [comment]
added comment to dnsapi/README.md
and to /RADME.md
sry am a bit of a newbie to git, thanks a lot for looking over it

File: dnsapi/dns_ddnss.sh [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+_ddnss_rest() {
+  method=$1
+  param="$2"
+  _debug param "$param"
+  url="$DDNSS_DNS_API?$param"
+  _debug url "$url"
+
+  # DDNSS uses GET to update domain info
+  if [ "$method" = "GET" ]; then
+    response="$(_get "$url" | sed -e :a -e 's/<[^>]*>//g;/</N;//ba' | tail -n 1)"
```
Comment (helbgd) [inline dnsapi/dns_ddnss.sh]: fix for Travis CI format check

File: dnsapi/dns_ddnss.sh [hunk 2]
```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env sh
+
+#Created by RaidenII, to use DuckDNS's API to add/remove text records
+#06/27/201
+#modified by helbgd @ 03/13/2018 to support ddnss.de
+#Please note: the Wildcard Feature must be turned on for the Host record
+#and the checkbox for TXT needs to be enabled
+
```
Comment (helbgd) [inline dnsapi/dns_ddnss.sh]: empty line deleted

File: dnsapi/dns_ddnss.sh [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+# Pass credentials before "acme.sh --issue --dns dns_ddnss ..."
+# --
+# export DDNSS_Token="aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+# --
+#
+
```
Comment (helbgd) [inline dnsapi/dns_ddnss.sh]: another empty line deleted

**Comment by helbgd** [comment]
Hello, would you please let me know what is missing to get merged ?

**Comment by Neilpang** [comment]
please  add the code and the doc in the same PR.

File: dnsapi/dns_ddnss.sh (hunk 3 above)
Comment (mod242) [reply dnsapi/dns_ddnss.sh]: @helbgd Just wanted to let you know ,that I modified your script very slightly and made a pull request that just got merged. Thank you very much for your great work (I hope it is ok for you that I took your script and modified it. Of course I left all the credits in there...)

**Pull request closed without merging**